docs/common/getting_started.md

```diff
@@ -160,14 +160,14 @@ Now, you should have three `.pem` files containing the key pair and the CA certi
 Install credentials onto the device:
 
 ```shell
-device_credentials_installer -d --ca *_ca.pem --ca-key *_prv.pem --coap --cmd-type at_shell
+python device_credentials_installer.py -d --ca *_ca.pem --ca-key *_prv.pem --coap --csv onboard.csv --cmd_type at
```

Comment:
If the python pakage is installed, as referenced on line 131 using `pip install nrfcloud-utils`, then the correct way to run this is
```
device_credentials_installer <params>
```